main.nf

```diff
@@ -254,11 +256,19 @@ if (params.library == 'classic'){
     library = '--UDGhalf'
 }
 
-if( ! nextflow.version.matches(">= 0.30") ){
+if( ! nextflow.version.matches(">= 19.04.0") ){
     println "Your version of Nextflow is too old, please update to Nextflow >= 0.30"
```

Comment:
Your error statement is wrong, even though condition is correct.

Reply:
Updated to use the version in manifest b95bc5e 
```
if( ! nextflow.version.matches(workflow.manifest.nextflowVersion) ){
    println "Your version of Nextflow is too old, please use Nextflow "+workflow.manifest.nextflowVersion.toString()
    exit(1)
}
```